litmus-portal/frontend/src/views/Community/GeoMap/CityMap.tsx

```diff
@@ -60,7 +59,15 @@ const CityMap: React.FC = () => {
           </Geographies>
           {mapData &&
             mapData.map(({ name, coordinates }) => (
-              <Marker key={`${name}_${coordinates}`} coordinates={coordinates}>
+              <Marker
+                key={`${name}_${coordinates}`}
+                coordinates={coordinates}
+                style={{
+                  default: { fill: '#5B44BA' },
+                  hover: { fill: '#2CCA8F' },
+                  pressed: { fill: '#5B44BA' },
+                }}
+              >
                 <g
                   stroke="#858CDD"
```

Comment:
use the `useTheme()` hook.

Reply:
@Saranya-jena @ishangupta-ds those colors are props for map we can not keet that in UseTheme().